agents/postgres/pgstatstatements/pgstatstatements.go

```diff
@@ -239,13 +240,14 @@ func makeBuckets(current, prev map[int64]*pgStatStatementsExtended, l *logrus.En
 			currentPSS.Tables = extractTables(currentPSS.Query, l)
 		}
 
+		fingerprint, _ := truncate.Query(currentPSS.Query)
```

Comment:
We do already truncate it in https://github.com/percona/pmm-agent/blob/PMM-5283-truncate-query-example-and-fingerprint/agents/postgres/pgstatstatements/stat_statements_cache.go#L128, so maybe we shouldn't do it there.

Reply:
Fixed.